recipes/opam/LICENSE

```diff
@@ -0,0 +1,198 @@
+Opam is distributed under the terms of the
```

Comment:
This file isn't needed as it's in the source tarball

Reply:
Removed, thanks!